src/main/java/edu/ucsb/ucsbcslas/DemoApplication.java

```diff
@@ -14,4 +19,13 @@ public static void main(String[] args) {
     SpringApplication.run(DemoApplication.class, args);
   }
 
+  // By default, the Jackson ObjectMapper is unable to properly serialize the Java LocalTime class.
```

Comment:
just wanted to call out this addition to DemoApplication. I was a bit hesitant adding any code to the DemoApplication class since it was previously pristine, but doing it this way avoids manually registering the module in every class that uses the ObjectMapper bean.